CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,66 @@
+# Code of Conduct
+
+We value the participation of every member of our community and want to ensure an that every contributer has an enjoyable and fulfilling experience. Accordingly, everyone who participates in the BIDS Starter Kit project is expected to show respect and courtesy to other community members at all time.
+
+Chris Gorgolewski, founder of the BIDS project, Chris Holdgraf, Dora Hermes, Elizabeth DuPre and Kirstie Whitaker, lead maintainers of the BIDS Starter Kit project, and contributors to the BIDS Starter Kit project are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

Comment:
> and contributors to the BIDS Starter Kit

Is this identifying us as contributors, or is it saying that all contributors to the project are _also_ dedicated to a harassment -free experience ? That phrasing is just a little unclear to me, personally !

Reply:
Is there a reason to keep the founder vs maintainers distinction ? The easiest fix I'd see would just be to list everyone's names, with no reference to their relationship to the project ! Not sure if there are strong feelings on keeping those titles, though.

Reply:
Yes, I think that's possibly a good solution. I think I was being overly cautious about not taking too much credit!! @chrisfilo - does @emdupre's suggestion sound ok for you?

Reply:
No strong feelings here. Just an empty husk. All I see is emptiness...

On a serious note - please do whatever you see fit. No need to set my name apart (and if you really want to set it apart just use my real first name - if you can spell it ;)

Reply:
Haha - that seems WAAAAY too ambitious.

Ok, how about this as a re-write:

> The lead maintainers of the BIDS Starter Kit (Chris Gorgolewski, Chris Holdgraf, Dora Hermes, Elizabeth DuPre and Kirstie Whitaker), and all contributors to the BIDS Starter Kit project are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.

Reply:
I might go a bit wild and link to each of those github profiles. Shout if you don't like that idea 😬

Reply:
I guess my one remaining question is: why not say 

> The lead maintainers of the BIDS Starter Kit (Chris Gorgolewski, Chris Holdgraf, Dora Hermes, Elizabeth DuPre and Kirstie Whitaker) are dedicated....

? I just am not sure that it makes sense to say "all contributors" are dedicated in the way that it makes sense to say all of your lab members are dedicated, since, at some point, the code of conduct will come into force when some contributor is not.

Reply:
OOOOOOH. I had *not* understood that was the point you were making. Got it.

I see your point. I have opinions about this phrasing in both directions, but lets go with your version for now (I'm dying on the gitter hill in #63 - no need to fight unnecessary battles!!)

Reply:
I'm totally open to a longer discussion / revisiting this ! Maybe this would be a good use of this gitter channel :smile: